hack/update_formula.bash

```diff
@@ -9,7 +9,7 @@ set -euo pipefail
 
 version=${version:-0.7.0}
 
-git pull
+#git pull
```

Comment:
removed this.  This repo was intended to just push to master.  If we want to create PR's we would be running this on branches that done exist in the remote.